projects/core/src/user/facade/user.service.spec.ts

```diff
@@ -17,6 +17,7 @@ import { UserRegisterFormData } from '../model/user.model';
 import * as fromStore from '../store/index';
 import { USER_FEATURE } from '../store/user-state';
 import { UserService } from './user.service';
+import { Consignment } from 'projects/backend/occ-client/typings/lib/models';
```

Comment:
You should not import files outside of project's folder (`projects/core`). It is probably also breaking tests.